### ISSUE TITLE
Blockquote and List don't support nesting

### ISSUE THREAD

**Issue by VHPaiva** [christianalfoni/markdown-to-react-components#18]
Blockquote and list have two problems working with nested elements.
If there are two line breaks, elements gets duplicated outside the blockquote;

```
List

1. Item

2. Item
```

The same occurs if other elements are nested, but the inner version returns as `undefined`;
```
> # Title
> Some text
>
> More Text
```

**Comment by christianalfoni** [comment]
Yeah, know about this bug, but have not found any time looking into it :(

**Comment by richardgorman** [comment]
I am having the same issue, when can we expect a fix to arrive?

**Comment by christianalfoni** [comment]
@richardgorman When you create a PR for it :)
